server/odc-service/src/main/java/com/oceanbase/odc/config/CommonSecurityProperties.java

```diff
@@ -121,6 +123,9 @@ public class CommonSecurityProperties {
     @Value("${odc.web.security.cors.allowedOrigins:*}")
     private List<String> corsAllowedOrigins;
 
+    @Value("${odc.web.security.auth-whitelist:}")
+    private List<String> securityAuthWhiteList;
```

Comment:
1. we may distinguish buitin whitelist and self defined whitelist, or the concept is confused due same configruation name and field name 'auth-whitelist'.

2. WhiteList or Whitelist will be okay either,  but if filed name 'WhiteList', the configuraiton item key may 'white-list'.

3. if the new configration is for cloud scenario only, we may not involved here

Reply:
Agree with points 1 and 2！For the point 3, it can indeed be done in other ways，i will close this pr.